nftables_test.go

```diff
@@ -2386,3 +2386,158 @@ func TestSet4(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestMasq(t *testing.T) {
+	tests := []struct {
+		name      string
+		chain     *nftables.Chain
+		want      [][]byte
+		masqExprs []expr.Any
+	}{
+		{
+			name: "Masquerada",
```

Comment:
Here and below, correct “Masquerada” to “Masquerade”

Reply:
done